cmake/templates/BoostConfig.cmake.in

```diff
@@ -39,6 +39,18 @@ if(NOT TARGET "Boost::boost")
         INTERFACE_COMPILE_DEFINITIONS "BOOST_ALL_NO_LIB=1"
     )
   endif()
+
+  if(BOOST_CONFIG_LINK_OPENSSL)
+    hunter_add_package(OpenSSL)
```

Comment:
If `websocketpp` use boost.asio with OpenSSL support (and this fact doesn't affect how Boost build) then it can be resolved by:

```cmake
hunter_add_package(Boost)
hunter_add_package(OpenSSL)

# find_*

target_link_libraries(foo PUBLIC Boost::boost ${OpenSSL_LIBRARIES})
```